Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN apk update --verbose \
  #TODO remove specifying respository once openjdk15 is in latest-stable branch
  && apk add --verbose --no-cache --repository 'http://dl-cdn.alpinelinux.org/alpine/edge/testing' \
     openjdk15 \
+ && apk add --verbose --no-cache --repository 'http://dl-cdn.alpinelinux.org/alpine/edge/community' \
+    shfmt \
```

Comment:
`shfmt` is on `community` not `main`
https://pkgs.alpinelinux.org/packages?name=shfmt&branch=edge